src/widgets/DisassemblyWidget.cpp

```diff
@@ -649,6 +650,49 @@ bool DisassemblyWidget::eventFilter(QObject *obj, QEvent *event)
         const QTextCursor& cursor = mDisasTextEdit->cursorForPosition(QPoint(mouseEvent->x(), mouseEvent->y()));
         jumpToOffsetUnderCursor(cursor);
 
+        return true;
+    } else if (event->type() == QEvent::ToolTip
+              &&  obj == mDisasTextEdit->viewport()) {
+
+        QHelpEvent * helpEvent = static_cast<QHelpEvent*>(event);
+
+        auto cursorForWord = mDisasTextEdit->cursorForPosition(helpEvent->pos());
+        cursorForWord.select(QTextCursor::WordUnderCursor);
+
+        RVA offsetFrom, offsetTo = readDisassemblyOffset( cursorForWord );
+        bool offsetsDiffer = false; //Do these 2 values differ?
+
+        QList<XrefDescription> refs = Core()->getXRefs( offsetTo, false, false);
```

Comment:
I think it should be `offsetFrom`, right? as this produce the references **from** this address
```suggestion
        QList<XrefDescription> refs = Core()->getXRefs(offsetFrom, false, false);
```

Reply:
I see your point. I will correct this and initialize the `offsetTo` with `RVA_INVALID` because basically at this point it is unknown.
```cpp
        RVA offsetFrom = readDisassemblyOffset(cursorForWord);
        RVA offsetTo = RVA_INVALID;
```